### PR TITLE
fixed missing prev/next on root texts

### PR DESCRIPTION
**Pull request by gradam**

### CODE REVIEW

File: client/elements/text/sc-text-page-selector.html [hunk 1]
```diff
@@ -221,6 +227,13 @@ <h3>[[lastError.statusText]]</h3>
                   if (this.responseData.translation) {
                       this.authorUid = this.responseData.translation.author_uid;
                   }
+                  if(this.translatedSutta) {
+                      this.next = this.translatedSutta.next;
+                      this.previous = this.translatedSutta.previous;
+                  } else {
```
Comment (TrebuhD) [inline client/elements/text/sc-text-page-selector.html]: what happens if `rootSutta` is `undefined`?

File: client/elements/text/sc-text-page-selector.html [hunk 2]
```diff
@@ -221,6 +227,13 @@ <h3>[[lastError.statusText]]</h3>
                   if (this.responseData.translation) {
                       this.authorUid = this.responseData.translation.author_uid;
                   }
+                  if(this.translatedSutta) {
```
Comment (TrebuhD) [inline client/elements/text/sc-text-page-selector.html]: missing space before `(`

File: client/elements/text/sc-text-page-selector.html (hunk 1 above)
Comment (gradam) [reply client/elements/text/sc-text-page-selector.html]: I think the situation where both, translatedSutta and rootSutta, is impossible. Am I missing something?

Comment (TrebuhD) [reply client/elements/text/sc-text-page-selector.html]: I wouldn't say it's impossible. It could easily happen in case of a server/database error.

**Pull request merged**